bazel/benchmark/component_benchmark/test/app.perf.spec.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {element, By} from 'protractor';
+import {runBenchmark} from '../../driver-utilities/index';
+
+describe('app performance', () => {
+  it('should update on button press', async () => {
+    await runBenchmark({
```

Comment:
This is more a comment on the component benchmarking tests I suppose, but we don't actually assert this does anything.

Reply:
added an actual assertion test

Reply:
👍  Obviously outside of the scope of this PR, but should the component benchmark also include a requirement of providing an assertion that verifies the `work` performs the expected action?  Then `run_benchmark` could one time run the assertion to make sure the test passes, and then perform the actual benchmarks as it currently does.